Readme to Crassus Page (#6)
susird/crassus-js: README.md

> #Crassus page :cactus:

Reviewer: ```suggestion
# Crassus page :cactus:
```

> ![alt-text](Captura de pantalla de 2020-10-25 22-06-42.png)

Reviewer: I think this image route isn't working because of the spaces in the name. Please consider renaming it, with a better name and dashes or underscore to separate words.

> This is the upgraded version of Crassus page, create with vanilla Javascript, HTML and CSS.

Reviewer: You can add the url to your first crassus, so the person who reads this can easily go to review it. 
```suggestion
This is the upgraded version of the first [Crassus page](https://github.com/susird/crassus-basic), created with vanilla Javascript, HTML and CSS.
```

This project was served using the  Live Server vscode extension. :satellite:

> ##Things I learned

Reviewer: ```suggestion
## Things I learned
```

⋅⋅* Develop an appropriate project structure.
⋅⋅* How to do a Json document, structure and syntax.
⋅⋅* Import and export data from external files.
⋅⋅* Render the different components of the app in an html node.

> ⋅⋅* Crate functions to create components from HTML

Reviewer: ```suggestion
⋅⋅* Crate functions to create components from HTML.
```
And all the items below that are missing the final point.

⋅⋅* Components are with Pascal case
⋅⋅* Use the VScode server extension
⋅⋅* Object destructuring
⋅⋅* Create a pull Request for each branch
⋅⋅* Create ul and li to iterate and interpolate in data elements.
⋅⋅* How to disappear an element with Js and css styles.
⋅⋅* Receive feedback and make code correction

> ⋅⋅* How to do a Read me. :page_facing_up:

Reviewer: The name is README.md

> ⋅⋅* How to update Master when changes are made to feature branch

Reviewer: ```suggestion
⋅⋅* How to update Master when changes are made to a feature branch
```

> ⋅⋅* Develop an appropriate project structure.
> ⋅⋅* How to do a Json document, structure and syntax.
> ⋅⋅* Import and export data from external files.
> ⋅⋅* Render the different components of the app in an html node.
> ⋅⋅* Crate functions to create components from HTML
> ⋅⋅* Components are with Pascal case
> ⋅⋅* Use the VScode server extension
> ⋅⋅* Object destructuring
> ⋅⋅* Create a pull Request for each branch
> ⋅⋅* Create ul and li to iterate and interpolate in data elements.
> ⋅⋅* How to disappear an element with Js and css styles.
> ⋅⋅* Receive feedback and make code correction
> ⋅⋅* How to do a Read me. :page_facing_up:
> ⋅⋅* How to update Master when changes are made to feature branch

Reviewer: ```suggestion
* Develop an appropriate project structure.
* How to do a Json document, structure and syntax.
* Import and export data from external files.
* Render the different components of the app in an html node.
* Crate functions to create components from HTML
* Components are with Pascal case
* Use the VScode server extension
* Object destructuring
* Create a pull Request for each branch
* Create ul and li to iterate and interpolate in data elements.
* How to disappear an element with Js and css styles.
* Receive feedback and make code correction
* How to do a Read me. :page_facing_up:
* How to update Master when changes are made to feature branch
```
Just putting the * will make the bullet point look good.

> ##Acknowledgment

Reviewer: 😭 ❤️

> ##Acknowledgment

Reviewer: ```suggestion
## Acknowledgment
```
